src/haversine.jl

```diff
@@ -9,6 +9,7 @@ The computed distance has the same units as that of the radius.
 struct Haversine{T<:Real} <: Metric
     radius::T
 end
+Haversine() = Haversine(6378137.0)
```

Comment:
Maybe it would be better to make this a `Float32` or an `Int32` to avoid promoting `Float32` values to `Float64` unnecessarily?